### PR TITLE
Swift51 support

### PR DESCRIPTION
**Pull request by sciabarracom**

### CODE REVIEW

**Comment by Andrea-Scuderi** [comment]
@sciabarracom I suspect it's required to add a test here [actionContainers](https://github.com/sciabarracom/openwhisk-runtime-swift/tree/swift51-support/tests/src/test/scala/runtime/actionContainers). The missing test should be Swift51CodableActionContainerTests.scala similar to Swift42CodableActionContainerTests.scala

**Comment by sciabarracom** [comment]
Yes you are right that it is missing one test set. The compilation problem however is related to the fact that references to test jars are outdated. Asking help to quickly find the reference to the current ones.

**Comment by sciabarracom** [comment]
added in the swift 51 pr

File: ansible/environments/local/group_vars/all [hunk 1]
```diff
@@ -59,6 +59,11 @@ runtimes_manifest:
       image:
         name: "action-swift-v4.2"
       deprecated: false
+    - kind: "swift:5.1"
+      default: true
```
Comment (csantanapr) [inline ansible/environments/local/group_vars/all]:
```suggestion
      default: false
```

Comment (Andrea-Scuderi) [reply ansible/environments/local/group_vars/all]: Good spot. What's the meaning of default? Do we want to leave `"swift:4.2"` as default?

Comment (csantanapr) [reply ansible/environments/local/group_vars/all]: Sure any, just one needs to be default. 
But first check that there are no tests that assume 3.1 is the default

**Pull request closed without merging**